src/hooks/common/AddCommaFunction.ts

```diff
@@ -0,0 +1,14 @@
+const AddCommaFunction = () => {
+  const AddComma = (index: number) => {
```

Comment:
함수형은 보통 카멜케이스로 하는거 같아요! 
```suggestion
const addCommaFunction = () => {
  const addComma = (index: number) => {
```